udata/tests/api/test_url_api.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import unittest
+
+from flask import url_for
+
+from . import APITestCase
+
+
+class CheckUrlAPITest(APITestCase):
+
+    def setUp(self):
+        # If you set up a Croquemort URL, don't forget to clean up the
+        # Redis database after each test. Otherwise the test against the
+        # delayed URL will not pass (existing entry already fetched).
+        self.CROQUEMORT_URL = self.create_app().config.get('CROQUEMORT_URL')
```

Comment:
I think it should be written:
```python
def serUp(self):
    super(CheckUrlAPITest, self).setUp()
    self.CROQUEMORT_URL = self.app.config.get('CROQUEMORT_URL')
    if self.CROQUEMORT_URL is None:
        raise unittest.SkipTest
```
It will avoid instanciating 2 or more apps and ensure parents setups are executed.